# Generated for NaoQi method ALSentinel.getBatteryLevel
# Get the battery level (0..5) 5: full 0: totally empty (never reached).

---
# the battery level (0..5): 5: full 0: totally empty (never reached)
int32 value


